docs/content/users/install/ddev-installation.md

```diff
@@ -2,7 +2,9 @@
 
 Once you’ve [installed a Docker provider](docker-installation.md), you’re ready to install DDEV!
 
-Installing and upgrading DDEV are nearly the same thing, because you're upgrading the `ddev` binary that talks with Docker. You can update this file like other software on your system, whether it’s with a package manager or traditional installer.
+## Upgrading
+
+Upgrading DDEV is basically the same process as installing it, because you’re upgrading the `ddev` binary that talks with Docker. You can update this file like other software on your system, whether it’s with a package manager or traditional installer.
```

Comment:
The problem is... this isn't actually true here. With brew it would be `brew upgrade`. With apt it would be `apt upgrade` as opposed to `apt install`.  

I do think this needs attention, but maybe it just needs more attention in the separate sections?

Reply:
What part of that overview isn’t true? The idea, I think, is that you’re updating an executable in the same way you first installed it.

Someone coming back to update needs a clear place to land, which is why Ben’s heading makes sense and my suggestion (B) of moving it further down the page defeats the purpose. I think shifting update notes exclusively into each tab would be a similar problem. Unless each tab had clear, separate _Installing_ and _Upgrading_ subheadings.

Or upgrading could get its own page?

Reply:
The idea is correct, but I think it implies the the process is always the same. So it's true that you're mostly just changing out a binary, but the execution, which is what most people care about, is slightly different, based on the flavor of the package manager, except in the case of the script. 

If you look at this in context, it also brings up upgrading before installation. Somehow we have to explain both. 

I was thinking maybe this could change to an overview of upgrading and installing without much change. This paragraph could explain both by getting across the idea that we're just swapping a binary, but the difference may be different with different package managers.

Reply:
To be clear, this is an important initiative, we just have to find the way to express it that gets people to the right place.